sagemaker-training-compiler/huggingface/tensorflow_multiple_gpu_single_node/language-modeling.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install transformers --upgrade
</pre><p><code>!pip install transformers==4.11.0</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
The training compiler DLCs use transformers v4.11.0 so that will ensure compatibility. However this could become outdated when newer versions are supported.

Reply:
This is also to avoid any breaking changes in future transformer releases. We can bump the versions up only when we complete testing with newer transformer libraries.